code/src/components/Header.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react'
+import styled from 'styled-components'
+
+import headerImage from '../header.jpg'
+
+export const Header = () => {
+
+    /* creates today's date */
+    const months = ['Jan', 'Feb', 'Mar', 'Apr', 'May', 'Jun', 'Jul', 'Aug', 'Sept', 'Oct', 'Nov', 'Dec'];
+    const date = new Date();
+    const mm = date.getMonth();
+    const month = months[mm];
+    const dd = date.getDate();
+    const weekday = date.toLocaleDateString('en-US', {weekday: 'long'});
+
+    return (
+        <HeaderContainer>
+            <Image src={headerImage} />
+                <HeaderText>MY TO-DO'S</HeaderText>
+                    <DateText>{weekday}, {month} {dd}</DateText>
+        </HeaderContainer>
+    )
+}
+
+const HeaderContainer = styled.section `
+display: flex;
+justify-content: center;
+align-items: center;
+position: relative;
+width: 100%;
+height: auto;
+`
```

Comment:
Quick fix for a (little bit) better look on desktop:
a `max-height` and `object-fit: cover;` ⭐